src/main/kotlin/uk/gov/dluhc/printapi/rds/entity/Certificate.kt

```diff
@@ -100,19 +98,23 @@ class Certificate(
     }
 
     @PrePersist
-    public fun prePersist() {
+    fun prePersist() {
```

Comment:
fun prePersist could be renamed to something like 'assignStatusBasedOnSomething' (not a blocker)

Reply:
Agree - but next PR is removing this method as the status change to the connected entities doesn't trigger the `@PreUpdate` annotation have refactored to set the status in response to request or status being added through the Certificate.